www/css/manage.css

```diff
@@ -65,10 +33,10 @@ form fieldset.storyFieldsetAction select {
 }
 .storylisting.functionTable tbody tr.event td:first-of-type {
     background:url('images/typeIndicatorEvent.png') no-repeat 0 0;
-}
+} /* No longer show News banner since every item is now news
```

Comment:
Should we just remove instead of commenting out

Reply:
I thought it might be helpful if someone goes looking for why there are only two types of banners and not a third